### PR TITLE
Connector.py class

### PR DESCRIPTION
**Pull request by markgllin**
Contains the following:

__init__(ip, port) --> initializes udp connection
send_slice(MemSlice) --> sends a MemSlice chunk to previously specified ip/port
get_slice() --> returns a MemSlice chunk from previously specified ip/port

### CODE REVIEW

File: Connector.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import socket
+import MemSlice
+
+class Connector():
+    BUF_SIZE = 1024
```
Comment (cwoz117) [inline Connector.py]: Should we take this and put it in a "constants" module?

File: Connector.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        Thread.__init__(self)
+        self.ip = ip
+        self.port = port
+        
+        # establish UDP connection
+        self.udpSocket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+
+    # send UDP datagram
+    def send_slice(MemSlice chunk):
+        self.udpSocket.sendto(chunk.to_bytestream(), (self.ip, self.port))
```
Comment (cwoz117) [inline Connector.py]: ``` (self.ip, self.port) ```

why the brackets?

File: Connector.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    def __init__(self, ip, port):
+        Thread.__init__(self)
+        self.ip = ip
+        self.port = port
+        
+        # establish UDP connection
+        self.udpSocket = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+
+    # send UDP datagram
```
Comment (cwoz117) [inline Connector.py]: trivial comments likely won't be needed

File: Connector.py (hunk 2 above)
Comment (markgllin) [inline Connector.py]: socket.sendto() defined as follows:
socket.sendto(data, address). Address is a tuple containing ip and port, i.e. address == (self.ip, self.port)

**Pull request merged**